resource_management_mobile/src/app/auth/login/login.page.html

```diff
@@ -0,0 +1,53 @@
+<!-- <ion-header>
+  <ion-toolbar>
+    <ion-title>login</ion-title>
+  </ion-toolbar>
+</ion-header> -->
+
+<ion-content [fullscreen]="true" >
+  <!-- <ion-header collapse="condense">
+    <ion-toolbar>
+      <ion-title size="large">login</ion-title>
+    </ion-toolbar>
+  </ion-header> -->
+  <section style="height: 100%;">
+    <div class="login_background">
+      <div class="login_header">
+        <img src="../../../assets/icon/logo.png" alt="log" style="width: 39%;">
+        <h2 class="header_text">GRAMENER</h2>
+        <label>Resource Management System</label>
+      </div>
+    </div>
+    <div class="login_model">
+        <div class="login_data_container">
+           <h4 class="admin_login">Admin</h4>
+           
+            <div class="item_list">
+              <form [formGroup]="loginForm"  novalidate autocomplete="true">
+              <ion-input label="User Name"  formControlName="userName" labelPlacement="floating" placeholder="Enter text"></ion-input>
+                <div *ngIf="loginForm.controls['userName'].invalid && (loginForm.controls['userName'].dirty || loginForm.controls['userName'].touched) || !onSubmit" class="alert alert-danger">
+                  <div *ngIf="loginForm.controls['userName'].errors?.['required'] ">
+                  Name is required.
+                  </div>
+                </div>
+              <ion-input label="Password"  formControlName="password" type={{type}}  id="password_parent" class="password_feiled" labelPlacement="floating" placeholder="Enter text"></ion-input>
+              <span (click)="passwordShow()" id="password_child">
```

Comment:
Restructure this with following URL:
https://forum.ionicframework.com/t/ionic-4-input-with-icon/165688
https://ionicframework.com/docs/api/input#clear-options

Reply:
Restructured the password section based on the shared resources.